amethyst_animation/src/material.rs

```diff
@@ -21,6 +22,15 @@ pub enum MaterialPrimitive {
     Offset((f32, f32), (f32, f32)),
 }
 
+use type_uuid::TypeUuid;
+use uuid::Uuid;
+
+impl TypeUuid for Sampler<MaterialPrimitive> {
+    const UUID: type_uuid::Bytes =
```

Comment:
Can we hide those behind a macro?

Reply:
unfortunately TypeUuid doesn't support generic types yet, so this is a workaround: https://github.com/randomPoison/type-uuid/issues/4